src/main/java/com/bettercloud/kadmin/kafka/BasicKafkaConsumerGroup.java

```diff
@@ -85,8 +87,19 @@ protected void init() {
         properties.put("metric.reporters", env.getProperty("metric.reporters", ""));
         properties.put("request.timeout.ms", env.getProperty("request.timeout.ms", Integer.class, 40 * 1000));
         properties.put("connections.max.idle.ms", env.getProperty("connections.max.idle.ms", Integer.class, 9 * 60 * 1000));
-        properties.put("security.protocol", env.getProperty("security.protocol", "PLAINTEXT"));
-
+        if(config.getSecurityProtocol().equals("SSL")) {
```

Comment:
Could you update `src/main/application.properties` to include a default value for this? I assume that you are setting it to `SSL`, but by default, it will be `null` causing a NPE on this line.

Reply:
absolutely! thanks for checking it